syntax = "proto3";

service ModelService {
  rpc StoreResults (ResultsRequest) returns (ResultsResponse);
}

message ResultsRequest {
  string image_url = 1; 
  repeated string class_labels = 2; 
  repeated float confidences = 3; 
}

message ResultsResponse {
  bool success = 1;
  string message = 2;
}
